index-de.html

```diff
@@ -195,7 +200,7 @@ <h3 id="commit">Commit</h3>
   <p>Ein vierter Fall ist, ein Commit mit <a href="#detached">detached HEAD</a>,
   wird weiter unten erklärt.</p>
 
-  <h3 id="checkout">Checkout</h3>
+  <h3 id="checkout">Checkout (Ausprobieren)</h3>
```

Comment:
@HybridDog I agree. However, "ausprobieren" is a valid translation of "checkout" and it describes the process of setting the stage and working directory to a given state (to try it out). I can't imagine a possible connection between "Ausstempeln" and git checkout. It just feels artificial. Could you explain it?